zScanner/Services/Storage/Model/DocumentDatabaseModel.swift

```diff
@@ -47,12 +47,12 @@ extension DocumentDatabaseModel {
         return DocumentDomainModel(
             id: id,
             folder: folder!.toDomainModel(),
-            type: type!.toDomainModel(),
+            type: type?.toDomainModel() ?? DocumentTypeDomainModel(id: "", name: ""),
             date: date,
             name: name,
             notes: notes,
             pages: pages.map({ $0.toDomainModel() }),
-            department: department!.toDomainModel()
+            department: department?.toDomainModel() ?? DepartmentDomainModel(id: "", name: "")
```

Comment:
I'm just curious, why did you make this change?

Reply:
That's a good point. I just realize that I shouldn't do it because each document has department and type. My bad.

Reply:
It should not happen. It might be because of some data inconsistency during development, it should not happen in production